WSLBlobNFS/wsl2_linux_script.sh

```diff
@@ -44,11 +124,12 @@ function install_samba ()
 function onetime_samba_setup ()
 {
     # Set username as the password
-    echo "Note: Samba password for $1 is same as the username. This makes it easier to mount shares."
+    vecho "Note: Samba password for $1 is same as the username. This makes it easier to mount shares."
     # remove "Added user $1" message
     echo -ne "$1\n$1" | tee - | smbpasswd -a -s $1
 
     # Add get quota script to the Samba global config
+    # To-do: Add a warning that quota is not supported for all the SMB shares.
     sed -i "/\[global\]/ a get quota command = $(dirname -- $0)/query_quota.sh" /etc/samba/smb.conf
```

Comment:
nit, put 'a' right after the pattern match string, so

sed -i "/\[global\]/a get quota command = $(dirname -- $0)/query_quota.sh" /etc/samba/smb.conf

also we can check before adding and if already exists don't add, o/w multiple runs might end up adding duplicate lines

Reply:
this command is not working, maybe the address selector query is not correct.
sed -i "/[global]/a get quota command = $(dirname -- $0)/query_quota.sh" /etc/samba/smb.conf